Description: Disable reference.md "Items_and_attributes_15" doctest, failing on armhf
 This passes on i386 amd64 arm64
 .
 See https://gist.github.com/infinity0/2f5452d4ee9deb2f1bacb8b39c987c9c
 for thread stacktrace from 1.10.0 when it hung instead of failed; eddyb from
 #rustc suggested this might be an LLVM issue.
 .
 Last checked this still affects 1.17
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/rust-lang/rust/issues/40260
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/doc/reference/src/items.md
+++ b/src/doc/reference/src/items.md
@@ -411,7 +411,7 @@
 in the same way as any other Rust function, except that they have the `extern`
 modifier.
 
-```rust
+```rust,ignore
 // Declares an extern fn, the ABI defaults to "C"
 extern fn new_i32() -> i32 { 0 }
 
